install/jakartaee/bin/sig-test_se11.map

```diff
@@ -199,3 +199,6 @@ jakarta.batch=2.0_se11
 # Jakarta EE Security API 2.0
 # (This is JSR-375)
 jakarta.security.enterprise=2.0_se11
+
+# javax.rmi
+javax.rmi=1.8_se11
```

Comment:
@scottmarlow  - we have concern on version here - "javax.rmi=1.8_se11" (usually where we have supplied 1.8, its the version of API)

Reply:
Good point, there really is no version that would make sense as javax.rmi was removed from the JDK and not added to Jakarta EE.